es_setup.sh

```diff
@@ -0,0 +1,126 @@
+npm install --save-dev elasticdump
```

Comment:
What are we trying to do here?

Reply:
I'm not sure where this came from. I had not typed this. Maybe it's the result of a different pulldown?

Reply:
This is Jake's elasticsearch dump script.  Only needed for local dev.  You probably put it in your designsafe code folder and accidentally tracked it with git.